### PR TITLE
odb: convert Finished LEF file to debug

### PR DESCRIPTION
**Pull request by oharboe**
less verbose output

### CODE REVIEW

**Comment by oharboe** [comment]
@maliberty Any reason not to convert all these to debug?

File: src/odb/src/lefin/lefin.cpp [hunk 1]
```diff
@@ -2230,7 +2230,7 @@ bool lefin::readLef(const char* lef_file)
   if (_master_cnt)
     _logger->info(utl::ODB, 225, "    Created {} library cells", _master_cnt);
 
-  _logger->info(utl::ODB, 226, "Finished LEF file:  {}", lef_file);
+  _logger->debug(utl::ODB, 226, "Finished LEF file:  {}", lef_file);
```
[the remark this answers is not in the document]
Comment (maliberty) [reply src/odb/src/lefin/lefin.cpp]: should be debugPrint

**Comment by maliberty** [comment]
> @maliberty Any reason not to convert all these to debug?

It gives scope to any warnings/errors during reading multiple files.

**Comment by oharboe** [comment]
> > @maliberty Any reason not to convert all these to debug?
> 
> It gives scope to any warnings/errors during reading multiple files.

At the cost of clutter for the common case without errors.... Classic.problem...

**Pull request closed without merging**